sparrow.yaml

```diff
@@ -0,0 +1,29 @@
+app-id: com.craigrow.sparrow
```

Comment:
craigrow.com does not seem to resolve. Please see the app id guidelines linked in the PR template.

Reply:
i can write in this field **io.github.craigrow.sparrow**?

Reply:
Where is the upstream of the project? code host site/web page etc.?

Reply:
Its desktop app

Reply:
Where should I indicate this?

Reply:
My source code on gitlab

Reply:
Add my project on github 
thi my link [https://github.com/Strozer2002/sparrow-linux/tree/main](https://github.com/Strozer2002/sparrow-linux/tree/main)

Reply:
> Add my project on github thi my link https://github.com/Strozer2002/sparrow-linux/tree/main

OK then you can use `io.github.Strozer2002.sparrow` as the app id

Reply:
current one is fine